src/StatsAPI.jl

```diff
@@ -32,6 +32,16 @@ function pairwise end
 # of the default definition.
 function pairwise! end
 
+"""
+    HypothesisTest
```

Comment:
How about calling this `AbstractHypothesisTest`? I think we do this in general for abstract types.

Reply:
I'm a huge fan of not putting `Abstract` if there is no foresight of the name without `Abstract` having a concrete implementation. Here, I don't see a package defining a concrete `HypothesisTest` type.

I'm also a fan of not putting `Abstract` when the name of the type clearly shows that it is "abstract" in the conceptual sense: a hypothesis test is a general/umbrella term, so it fits for an abstract type.

Reply:
> I think we do this in general for abstract types.

Not for `RegressionModel` nor `StatisticalModel` which are the only other abstract types in this package.